mesonbuild/wrap/wrap.py

```diff
@@ -217,6 +220,11 @@ def get(self, key: str) -> str:
         except KeyError:
             raise WrapException(f'Missing key {key!r} in {self.basename}')
 
+    def update_hash_cache(self) -> None:
+        if self.has_wrap and os.path.isdir(self.directory) :
```

Comment:
When would it not be a directory? Only when the PR has a bug in it. :(

It is missing the prefixed `subprojects/` part as it turns out.

Reply:
I've removed this check; it should no longer be necessary.